.travis.yml

```diff
@@ -13,4 +13,4 @@ install:
   - travis_retry composer install
 
 script:
-  - phpunit
\ No newline at end of file
+  - phpunit -c phpunit.xml --tap --coverage-text
```

Comment:
I agree with this. We can certainly set up a single step for running coverage only once. :+1: